haystack/pipeline.py

```diff
@@ -142,14 +142,46 @@ def draw(self, path: Path = Path("pipeline.png")):
         graphviz.draw(path)
 
     @classmethod
-    def load_from_yaml(cls, path: Path, pipeline_name: Optional[str] = None):
+    def load_from_yaml(cls, path: Path, pipeline_name: Optional[str] = None, overwrite_with_env_variables: bool = True):
         """
         Load Pipeline from a YAML file defining the individual components and how they're tied together to form
         a Pipeline. A single YAML can declare multiple Pipelines, in which case an explicit `pipeline_name` must
         be passed.
 
+        Here's a sample configuration:
+
+        version: '0.7'
```

Comment:
We need to format the code snippet by adding a "|" in every line of code. See: https://github.com/deepset-ai/haystack/blob/1318b55eecf057c76e2758060736174a678e1e18/haystack/retriever/dense.py#L56